drivers/usb/udc/Kconfig.dwc2

```diff
@@ -48,3 +48,11 @@ config UDC_DWC2_USBHS_VBUS_READY_TIMEOUT
 	  UDC DWC2 USBHS VBUS ready event timeout. If the VBUS is not ready
 	  and the Nordic USBHS controller is used, the udc_enable() is
 	  blocked for this amount of time. Set it to zero to wait forever.
+
+config UDC_DWC2_PREFER_8BIT_UTMI
+	bool "Prefer 8-bit UTMI+ PHY interface width"
+	depends on UDC_DWC2
+	help
+	  UDC DWC2 may support both 8- and 16-bit UTMI+ PHY interface widths.
+	  Enable this config to prefer 8-bit interface. Otherwise 16-bit interface
+	  will be used.
```

Comment:
I can not see any justification why 8-bit should be preferred. And Kconfig is just the wrong place to configure it.

Reply:
If there is UTMI+ PHY capable of both 8-bit and 16-bit operation connected to DWC2 controller then there is very little need to use 8-bit (clocking is different between the two, so there hardware behavior changes slightly).

However, it is also possible that there is only 8-bit capable UTMI+ PHY connected to DWC2 controller that supports both 8-bit and 16-bit. In such case this is a killer feature (because 16-bit simply won't work).